RPS.js

```diff
@@ -154,20 +148,16 @@ var playerScore = 0;
 var cpuScore = 0;
 const playerScoreElement = document.querySelector('#player-score');
 const cpuScoreElement = document.querySelector('#cpu-score');
-function playerIncrementScore (score, scoreElement) {    
+function playerIncrementScore () {    
     console.log('The function playerIncrementScore() is now running....');
     playerScore++;
     playerScoreElement.innerHTML = `Score: ${playerScore}`;
 }
-function cpuIncrementScore (score, scoreElement) {
+function cpuIncrementScore () {
     console.log('The function cpuIncrementScore() is now running....');
     cpuScore++;
     cpuScoreElement.innerHTML = `Score: ${cpuScore}`;
 }
-function playerCPUDraw(playerScore, cpuScore) {
```

Comment:
I decided to remove this function, since nothing needs to happen if there's a draw. It's implicit, but sometimes explicit is good too, like what you did here